files/syncfusion-javascript/update.json

```diff
@@ -3,9 +3,7 @@
     "name": "syncfusion-javascript",
     "repo": "syncfusion/JavaScript-Widgets",
     "files": {
-      "basePath":"JavaScript-Widgets",
-      
-    "exclude": ["ej/ej.widgets.all.min.js","**/*.json",".npmignore",
-                "**/*.md"]
+       "include": ["./Content/**/*","./Scripts/**/*"], 
+       "exclude": ["ej/ej.widgets.all.min.js"]
```

Comment:
Hi @megawac,

Sorry for the trouble. We have excluded the same file from our repository. So I have removed the exclude block from this json and update the pull request here. 

https://github.com/jsdelivr/jsdelivr/pull/14217/commits/22cfa3a61d5f27da038dfd41b54246f3386c103f